doc/direct_credit.md

```diff
@@ -15,7 +15,10 @@ $customerCredit->addPaymentInfo('firstPayment', array(
     'debtorAccountIBAN'       => 'FI1350001540000056',
     'debtorAgentBIC'          => 'PSSTFRPPMON',
 ));
+// Add/Set batch booking option, you can pass boolean value as per your requirement
+$customerCredit->setBatchBooking(true);
 // Add a Single Transaction to the named payment
+// `amount` should be in cents
```

Comment:
Can you move this at the end of the line `'amount' => 500` I think this will make this more clear and visible